### PR TITLE
SDKS-5393. Add type definitions

### PR DESCRIPTION
**Pull request by emmaz90**
# JS SDK
Added Type definitions

## Extra Notes

### CODE REVIEW

**Review by mmelograno — APPROVED** [review]

File: types/splitio.d.ts [hunk 1]
```diff
@@ -1210,6 +1236,52 @@ declare namespace SplitIO {
...
+   */
+  interface IBrowserClient extends IClient {
+    /**
+     * Add an attribute to client's in memory attributes storage
+     * 
+     * @param {String} attributeName Attrinute name
+     * @param {AttributeType} attributeValue Attribute value
+     * @returns {boolean} true if the attribute was stored and false otherways
+     */
+    setAttribute(attributeName: String, attributeValue: AttributeType): boolean,
```
Comment (EmilianoSanchez) [inline types/splitio.d.ts:1252]: `: String` --> `: string`. Check also in lines 1259 and 1266

File: ts-tests/index.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@
  */
 
 import { SplitFactory } from '@splitsoftware/splitio';
+import { Attributes, AttributeType } from '../types/splitio';
```
Comment (EmilianoSanchez) [inline ts-tests/index.ts:15]: Emma, I think you can remove this line and replace the occurrences of `Attributes` with `SplitIO.Attributes` in the code. Same for `AttributeType`

File: ts-tests/index.ts [hunk 3]
```diff
@@ -195,6 +199,8 @@ manager = SDK.manager();
 // Today async clients are only possible on Node. Shared client creation not available here.
 asyncClient = AsyncSDK.client();
 asyncManager = AsyncSDK.manager();
+// Browser client for attributes binding
+browserClient = BrowserSDK.client();
```
Comment (EmilianoSanchez) [inline ts-tests/index.ts:203]: add `browserClient = BrowserSDK.client('a customer key');` in a new line, to check that the other `client` method signature also returns the same client type.

**Review by EmilianoSanchez — APPROVED** [review]
LGTM

**Pull request merged**